sound/soc/sof/intel/hda-dsp.c

```diff
@@ -704,6 +704,11 @@ static int hda_resume(struct snd_sof_dev *sdev, bool runtime_resume)
 	hda_dsp_ctrl_ppcap_enable(sdev, true);
 	hda_dsp_ctrl_ppcap_int_enable(sdev, true);
 
+	/* turn off display power as link access is done.
+	display codec driver will turn On for pcm ops */
+	if(runtime_resume)
```

Comment:
@sathya-nujella With the recent patch "ASoC: SOF: Intel: hda: use hdac_ext fine-grained link management", it might now be possible to do. We do need to do quite a bit of testing as there has been so many issues with this in the past. There's another recent patch to i915 driver "drm/i915/audio: set HDA link parameters in driver" (in review, not yet merged) that will help in the corner cases.

Reply:
sure @kv2019i .
I see your patch: https://patchwork.kernel.org/project/intel-gfx/patch/20210212150838.2568483-1-kai.vehmanen@linux.intel.com/.. thank you!  Seems not directly related to this issue, but it is good we have no dependency on Coreboot setting. Thank you for this. Will take it Chrome kernel.

As your PR#2762 also address the current issue of driver not going to PC10, will follow up in that PR. Thanks!